src/App.tsx

```diff
@@ -1,16 +1,24 @@
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import { Movie } from './types/Movie';
+
+export const App: React.FC = () => {
+  const [newMovies, setNewMovies] = useState(moviesFromServer);
+
+  const addNewMovie = (movie: Movie) => {
+    setNewMovies((curMovies) => [...curMovies, movie]);
+  };
 
-export const App = () => {
   return (
     <div className="page">
       <div className="page-content">
-        <MoviesList movies={moviesFromServer} />
+        <MoviesList movies={newMovies} />
       </div>
       <div className="sidebar">
-        <NewMovie /* onAdd={(movie) => {}} */ />
+        <NewMovie onAdd={addNewMovie} />
```

Comment:
```suggestion
        <NewMovie onAdd={addNewMovie} key={count} />
```
Adjust counter here. It will force re-render your component with default values and no errors.